pkg/plugin/plugin.go

```diff
@@ -109,9 +114,50 @@ func loadNetConf(bytes []byte) (*netConf, error) {
 		return nil, fmt.Errorf("failed to load netconf: %v", err)
 	}
 
+	// default config file dir paths
+	confdirs := []string{"/etc/kubernetes/cni/net.d/ovs.d/ovs.conf", "/etc/cni/net.d/ovs.d/ovs.conf"}
```

Comment:
Could we make this a constant on the top of the file? For better visibility.

Reply:
this is a slice of string type, can't make it as a constant. just moved it into a method.